distribution/lib/Standard/Base/0.0.0-dev/src/Data/Filter_Condition.enso

```diff
@@ -141,7 +143,16 @@ type Filter_Condition
          Using Columns can be particularly useful for Database operations, as
          uploading a temporary table and using its column for an `Is_In` check
          will likely be faster than using the vector directly.
-    Is_In values:Vector|Any action:Filter_Action=Filter_Action.Keep
+    Is_In values=[] action:Filter_Action=Filter_Action.Keep
+
+    ## PRIVATE
+       Resolves a possibly auto-scoped value to a concrete value.
+    resolve_auto_scoped : Any -> Any
```

Comment:
Please use `private` keyword.

Reply:
Can't in this case as used across different libraries but yes will do.